mm2src/mm2_event_stream/src/event.rs

```diff
@@ -0,0 +1,47 @@
+use serde_json::Value as Json;
+
+// Note `Event` shouldn't be `Clone`able, but rather Arc/Rc wrapped and then shared.
+// This is only for testing.
+/// Multi-purpose/generic event type that can easily be used over the event streaming
+#[cfg_attr(any(test, target_arch = "wasm32"), derive(Clone, Debug, PartialEq))]
+#[derive(Default)]
+pub struct Event {
+    /// The type of the event (balance, network, swap, etc...).
+    event_type: String,
```

Comment:
event_id is a String although it has a structure like "TASK:{task_id}" or "FEE_ESTIMATION:{ticker}".
But all event id constructions are scattered over the code - so it's easy to overlook which events we support, 
also the format is not checked - easy to make a mistake.
Maybe we could make event_id as a enum with the deser feature? It would be easier to maintain all event and document them for the GUI